src/ll/bit.rs

```diff
@@ -205,6 +205,26 @@ pub unsafe fn not(mut wp: *mut Limb, mut xp: *const Limb, n: i32) {
 }
 
 /**
+ * Computes the two's complement of the `xs` least significant words
+ * of `xp`. The result is stored the result in `wp`, and a carry is
+ * returned, if there is one.
+ */
+pub unsafe fn twos_complement(mut wp: *mut Limb, mut xp: *mut Limb, xs: i32) -> Option<Limb> {
```

Comment:
Seems a bit of waste to return `Option<Limb>` here when just using a `Limb` will suffice. Since it can only be 1 or 0.